src/GLFW.jl

```diff
@@ -22,8 +22,23 @@ else
 	error("GLFW $VERSION is not supported")
 end
 
+immutable GLFWError <: Exception
+    code::ErrorCode
```

Comment:
terrible things! :D So yeah, we should include an unknown field and then redirect to that.